app/src/main/java/com/example/parkingsystem/mvp/presenter/ParkingPresenter.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.parkingsystem.mvp.presenter
+
+import com.example.parkingsystem.mvp.contract.ParkingContract
+
+class ParkingPresenter(
+    private val model: ParkingContract.Model,
+    private val view: ParkingContract.View
+) : ParkingContract.Presenter {
+    override fun onParkingSizeButtonPressed() {
+        view.showParkingSizeToast(model.getParkingSize())
+    }
+
+}
```

Comment:
to avoid this error add an empty line at the end of the file

Reply:
Done 👍